src/helpers/get-pull-requests.ts

```diff
@@ -1,104 +1,112 @@
 import { CURRENT_DAY, DAYS_TO_INACTIVE, ONE_DAY } from "@/config"
 
-import { PullRequests, PR } from "../types/pull_requests"
+import { PullRequests, PR, PrNodes } from "../types/pull_requests"
 
 export const getPullRequests = ({
     data,
     username
 }: {
-    data: any
+    data: PrNodes[]
     username: string
 }): PullRequests => {
-    const openPRsData = data.pullRequests.edges.filter(
-        (pr: any) => pr.node.closed === false
-    )
-    const openPRs: PR[] = openPRsData.map((pr: any) => ({
-        totalComments: pr.node.totalCommentsCount,
+    data = data !== undefined ? data : []
+
+    const openPRsData = data?.filter((pr) => pr?.pullRequest?.closed === false)
```

Comment:
not sure about these optional chaining. can't we do a check for it the data is null and return. empty array maybe??